commands/FBDebugCommands.py

```diff
@@ -189,6 +191,88 @@ def run(self, arguments, options):
     fb.evaluateEffect('[[UIApplication sharedApplication] performSelector:@selector(_performMemoryWarning)]')
 
 
+def switchBreakpointState(expression,on):
+
+  expression_pattern = re.compile(r'{}'.format(expression),re.I)
+
+  target = lldb.debugger.GetSelectedTarget()
+  for breakpoint in target.breakpoint_iter():
+    if breakpoint.IsEnabled() != on and (expression_pattern.search(str(breakpoint))):
+      print str(breakpoint)
+      breakpoint.SetEnabled(on)      
+    for location in breakpoint:
+      if location.IsEnabled() != on and (expression_pattern.search(str(location)) or expression == hex(location.GetAddress()) ):
+        print str(location)
+        location.SetEnabled(on)
+
+class FBMethodBreakpointEnableCommand(fb.FBCommand):
+  def name(self):
+    return 'benable'
+
+  def description(self):
+    return """
+    Enable a set of breakpoints for a regular expression
+
+    Examples:
+
+          #use `benable disabled` to switch all breakpoints to `enable`
+          benable disabled
```

Comment:
I don't think this should be documented. A builtin command already does this: `breakpoint enable` (or `br en` for short).